MediaRecorder.bs

```diff
@@ -257,6 +256,8 @@ interface MediaRecorder : EventTarget {
 
     </ol>
     </li>
+    <li><a>Fire an event</a> named <a>start</a> at <var>target</var>.</li>
```

Comment:
So, normally, [in parallel](https://html.spec.whatwg.org/#in-parallel) steps describe steps to be performed in sequence on a background thread. Unfortunately, the approach taken in this method strays from that, in that it describes things that happen later, after the method has returned. That makes it unclear when this event fires.

Short of tearing this structure apart, and moving things to sections outside `start()`, I suggest we instead modify the first step to happen instantly by removing the "Once". Something like:

*"Start gathering data from all of the stream's tracks into a Blob blob, then queue a task, using the DOM manipulation task source, to fire an event named start at target, and then return from this method."*